TESTS/psa/entropy_inject/main.cpp

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
Critical: the commits that import-and-adapt entropy injection tests from work in progress then the final version of the unit tests in Mbed Crypto should reference the precise version that they are importing.

Reply:
Hi @gilles-peskine-arm the APIs should not change. so the tests remains relevant. 
and api changes can always happen regardless of this fact

Reply:
Because this is only test code, I'm willing to downgrade this to “major”. But in the future, please trace the history of code the way medical implants _should_ be traced, not the way medical implants _are_ traced.